.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": "x64-Debug"
+}
```

Comment:
it seems we're pushing .vs folder . That shouldn't be happening. I'll update the .gitignore and remove those files from versioning.

Reply:
Okay, let me know if you need help with that, I won't touch that for now.